### DISCUSSION TITLE
Who is the host of this year’s Academy Awards? 🎥🏆

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#287] [Galaxy Trivia]
**Hint:** JK Live! 📺

**Comment by thuongtruong1009** [comment]
The host of the Academy Awards (also known as the Oscars) can vary from year to year, and sometimes there is no official host. In recent years, the Oscars have not had a traditional host and instead have featured a rotating group of presenters and performers. If you are looking for information on the specific host of a particular year, I would suggest searching for news articles or official announcements from that time.

**Comment by ParanoidUser** [comment]
Does anyone know who hosted this year's Academy Awards? @darko5r, @awesomelewis2007, @anggerpr,  @KelechiOdom10, @jso4342
Go Go Power Rangers! 🤖

**Comment by 0x4248** [comment]
Regina Hall, Amy Schumer and Wanda Sykes

**Reply by 0x4248** [reply]
from https://en.wikipedia.org/wiki/94th_Academy_Awards

**Comment by ParanoidUser** [comment]
@live627, @MarieCamillePetit, @muink, @LauraBigoni, @PavelLaptev, @yt605155624 anyone watched the Oscars last Sunday? 😏

**Reply by live627** [reply]
haha nope

**Comment by Mithun1508** [comment]
Comedian and late-night host Jimmy Kimmel is returning once again to host this year's Oscars, marking the third time he's hosted the Academy Awards. Kimmel previously hosted the ceremony back in 2017 and 2018

**Reply by ParanoidUser** [reply]
Bingo! 👏👏 Congratulations on your well deserved [Galaxy Brain](https://github.com/Mithun1508?tab=achievements&achievement=galaxy-brain) badge. Continue to unlock the next badge levels (Bronze for 8 answers), and don't hesitate to post your own questions to the community. 😊🎉
